core/src/main/java/nl/nn/adapterframework/jdbc/JdbcQuerySenderBase.java

```diff
@@ -201,22 +202,40 @@ protected ResultSet getReturnedColumns(String[] columns, PreparedStatement st) t
 	@Override
 	protected String sendMessage(Connection connection, String correlationID, String message, ParameterResolutionContext prc) throws SenderException, TimeOutException {
 		PreparedStatement statement=null;
-		ParameterList newParamList = new ParameterList();
+		ParameterList newParameterList = new ParameterList();
 		if (paramList != null) {
-			newParamList = (ParameterList) paramList.clone();
+			newParameterList = (ParameterList) paramList.clone();
 		}
+		String newMessage;
 		if (isUseNamedParams()) {
-			message = adjustParamList(newParamList, message);
+			newMessage = adjustParameterList(newParameterList, message);
+		} else {
+			newMessage = message;
 		}
 		try {
 			boolean updateBlob="updateBlob".equalsIgnoreCase(getQueryType());
 			boolean updateClob="updateClob".equalsIgnoreCase(getQueryType());
 			log.debug(getLogPrefix() + "obtaining prepared statement to execute");
-			statement = getStatement(connection, correlationID, message, updateBlob||updateClob);
+			statement = getStatement(connection, correlationID, newMessage, updateBlob||updateClob);
 			log.debug(getLogPrefix() + "obtained prepared statement to execute");
 			statement.setQueryTimeout(getTimeout());
-			if (prc != null && paramList != null) {
-				applyParameters(statement, prc.getValues(newParamList));
+			
+			//TODO: move if-block below to DbmsSupport?
+			if ((updateBlob || updateClob) && OracleDbmsSupport.dbmsName.equalsIgnoreCase(getSqlDialect()) && !OracleDbmsSupport.dbmsName.equalsIgnoreCase(getDbmsSupport().getDbmsName())) {
```

Comment:
Het is niet goed als JdbcQuerySenderBase afhankelijk is van OracleDbmsSupport.
Inderdaad verhuizen naar DbmsSupport

Reply:
https://github.com/ibissource/iaf/pull/251/commits/c0596ae555c3a3eb1a2ab240df6445fccff77606